### PR TITLE
feature: add global success message when settings address as default

### LINKED ISSUE #13719: Missing successful global message when user sets additional address as default
**Describe the bug**
- Screen reader users wouldn't be able to tell if they have changed an address as the new default.

**Tell us the version of Spartacus**
- Library version: 4.0.0

**To Reproduce**
1. Go to My Account > Address Book
2. Add 2 different addresses or more.
3. Open any Screen Reader (VO, NVDA, JAWS).
4. Switch addresses from additional to new defaults.
5. Listen to Screen Reader feedback
6. Hear the absence of confirmation a change has happened.

**Expected behavior**
- Screen reader users will hear a successful confirmation message as they set an address as new default. E.g _"Additional address successfully set as new default"._
- For `MSG_TYPE_CONFIRMATION` use `aria-live="polite" aria-atomic="true"`

**Screenshots**

Message suggestion in the screenshot below, please validate phrasing with @Manurod2018 or @vekterdude

a2021-09-14_20-51-14

### PR DESCRIPTION
**Pull request by RadhepS**
closes: https://github.com/SAP/spartacus/issues/13719

### CODE REVIEW

**Review by dydome — DISMISSED** [review]

File: projects/storefrontlib/cms-components/myaccount/address-book/address-book.component.html [hunk 1]
```diff
@@ -29,7 +29,7 @@
           [fitToContainer]="true"
           [content]="getCardContent(address) | async"
           (editCard)="editAddressButtonHandle(address)"
-          (setDefaultCard)="setAddressAsDefault(address.id)"
+          (setDefaultCard)="setAddressAsDefault(address)"
```
Comment (Matejk00) [inline projects/storefrontlib/cms-components/myaccount/address-book/address-book.component.html:33]: Please mention about this change in migration doc.

File: projects/storefrontlib/cms-components/myaccount/address-book/address-book.component.ts [hunk 2]
```diff
@@ -106,8 +107,12 @@ export class AddressBookComponent implements OnInit {
     );
   }
 
-  setAddressAsDefault(addressId: string): void {
-    this.service.setAddressAsDefault(addressId);
+  setAddressAsDefault(address: Address): void {
```
Comment (Matejk00) [inline projects/storefrontlib/cms-components/myaccount/address-book/address-book.component.ts:118]: the parameter has been changed, please add a note to migration doc

File: projects/storefrontlib/cms-components/myaccount/address-book/address-book.component.html (hunk 1 above)
Comment (RadhepS) [reply projects/storefrontlib/cms-components/myaccount/address-book/address-book.component.html:33]: Added

File: projects/storefrontlib/cms-components/myaccount/address-book/address-book.component.ts (hunk 2 above)
Comment (RadhepS) [reply projects/storefrontlib/cms-components/myaccount/address-book/address-book.component.ts:118]: Added

**Review by dydome — DISMISSED** [review]

**Review by Matejk00 — DISMISSED** [review]

**Review by dydome — DISMISSED** [review]

**Review by dydome — DISMISSED** [review]

File: docs/migration/5_0.md [hunk 3]
```diff
@@ -54,6 +54,10 @@ under the section `headerDisplayOnly`.
 
 ### AddressFormComponent
 
+#### AddressBookComponent
+
+- Changed `setAddressAsDefault` method to accept type `Address` instead of type `string`
```
Comment (psmul) [inline docs/migration/5_0.md]:
```suggestion
- Changed `setAddressAsDefault` method parameter type`string` to `Address`.
```

**Review by psmul — CHANGES_REQUESTED** [review]

**Review by psmul — DISMISSED** [review]

**Review by psmul — APPROVED** [review]

**Review by hackergil — APPROVED** [review]

**Pull request merged**